test/integration/targets/user/tasks/main.yml

```diff
@@ -1135,3 +1135,336 @@
   when: ansible_facts.system in ['Linux']
   tags:
     - user_test_local_mode
+
```

Comment:
These tests are great. This `main.yml` file is a rather huge and I plan to break it up at some point. Could you move these tests into a separate tasks file and use `import_tasks` to make my future work a bit easier?

Reply:
Done in https://github.com/ansible/ansible/pull/72022/commits/d5c473e34986277a0cd35390608e4cdca2a1eb5f